modules/ROOT/pages/managing-instances/instance-actions.adoc

```diff
@@ -58,20 +58,13 @@ image::instancedetails.png[]
 == Resize an instance
 
 Resizing an instance means changing the memory, CPU and storage size.
+Note that this option is not available for Free instances.
 
 // label:AuraDB-Professional[]
 // label:AuraDB-Enterprise[]
 // label:AuraDB-Business-Critical[]
 
-You can change the size of an existing instance using the *Resize* action.
-
-To resize an instance:
-
-. Select the *resize* button on the instance you want to resize.
-. Select *Resize* from the resulting menu.
-. Select the new size you want your instance to be.
-. Tick the *I accept* checkbox and select *Resize*.
-
+You can change the size of an existing instance using the *Resize* button on the instance you want to resize..
```

Comment:
I like what you did here, simplifying this. Maybe we can say, "Use the Resize button to change the size of an existing instance." @AlexicaWright

Reply:
That assumes that the user wants to resize, which we can't know.